compiler/compile/OMRSymbolReferenceTable.hpp

```diff
@@ -240,8 +240,8 @@ class SymbolReferenceTable
    TR::SymbolReference * getSymRef(CommonNonhelperSymbol i);
    TR::SymbolReference * getSymRef(int32_t i) { return baseArray.element(i); }
 
-   TR::SymbolReference * createRuntimeHelper(TR_RuntimeHelper, bool = false, bool = false, bool preservesAllRegisters = false);
```

Comment:
Can you explain the rational in removing the default parameter values? It would also be more ideal to have names for the bool parameters in the header so it is easy to find out what the different params are and what they mean without having to drill into the cpp.